sky/clouds/aws.py

```diff
@@ -368,9 +375,34 @@ def check_credentials(self) -> Tuple[bool, Optional[str]]:
         except exceptions.CloudUserIdentityError as e:
             return False, str(e)
 
+        hints = None
+        if self._is_current_identity_sso():
+            hints = (
+                'AWS SSO is set. '
+                'It will work if you use AWS only, but will cause problems if you want to '
```

Comment:
Minor: I have both the credentials file and SSO, and the line `To set up static credentials, try: aws configure` could be a bit surprising (since I already have it).

How about:
```
To ensure multiple clouds work correctly, please use non-SSO credentials (e.g., ~/.aws/credentials).
```

Reply:
Good catch! Handled the special cases. Thanks!